roles/common/tasks/networking.yml

```diff
@@ -13,6 +13,36 @@
   with_items: network_interfaces_d
   tags: ['prereboot']
 
+- name: set default network gateway
+  set_fact: dfl_nw_gw="{{ ansible_default_ipv4.network.split('.')[0] }}.{{ ansible_default_ipv4.network.split('.')[1] }}.{{ansible_default_ipv4.network.split('.')[2] }}.{{ (ansible_default_ipv4.network.split('.')[3] | int) + 1 }}"
+  when: update_routes|default('false')|bool
+
+- name: add static route for multi subnet
+  command: "/sbin/ip route replace {{ item }} via {{ dfl_nw_gw }}"
+  when: (not (ansible_default_ipv4.network ==  ( item | ipaddr('network') ))) and (update_routes|default('false')|bool)
+  with_items: "{{ undercloud_cidr }}"
```

Comment:
maybe we could expand {{ undercloud_cidr }} to include optional information such as gateway, that way we don't have to try and guess at it.  could also flag whether or not it needs a static route set in the dict.